.gitignore

```diff
@@ -0,0 +1,3 @@
+/.env

```

Comment:
Please change "/.env" into ".env", otherwise the env file still will upload to github